workers/fund_public_goods/db/entities.py

```diff
@@ -123,8 +123,6 @@ class Projects(BaseModel):
     website: str
     logo: Optional[str] = None
     twitter: Optional[str] = None
-    keywords: list[str] = []
```

Comment:
This file is generated, so this will be reverted with a new run of `yarn codegen`